test/testdata/generate.sh

```diff
@@ -98,32 +98,23 @@ function scaffold_test_project {
     $kb create api --group foo --version v1 --kind Bar --controller=true --resource=true --make=false
     $kb create api --group fiz --version v1 --kind Bar --controller=true --resource=true --make=false
 
-    if [ $project == "project-v3-multigroup" ] || [ $project == "project-v4-multigroup" ]; then
+    if [ $project == "project-v4-multigroup" ] || [ $project == "project-v4-multigroup-with-deploy-image" ] ; then
       $kb create api --version v1 --kind Lakers --controller=true --resource=true --make=false
       $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation
     fi
-
-    # Call ALL optional plugins within multigroup layout to ensure that they can work within
-    header_text 'Creating Memcached API with deploy-image plugin ...'
-    $kb create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false
```

Comment:
@camilamacedo86 why remove this? Even if its v4, we still need to test if we scaffold out 2 APIs. Or am I missing the code for it somewhere out of the  diff files?

Reply:
See that this code was duplicated and it was not called only for project-v4-withdeploy-image sample
Therefore, this code was scaffolding the APIs using deploy image for the project-v4 sample when should not.

We should only to do this scaffold for the samples which specific uses the deploy-image.